src/components/Client/RESTClient.js

```diff
@@ -269,6 +269,31 @@ class RESTClient {
         return response.data;
     }
 
+    async claimRecord(recordID, userToken) {
```

Comment:
Please add the JSdoc comments for these two methods.
I'm also realize i forgot to add them for edit and create publications.